src/softerllama/modeling_softerllama.py

```diff
@@ -51,8 +51,9 @@ def __init__(self, config: SofterLlamaConfig, layer_idx: Optional[int] = None):
                 "when creating this class."
             )
 
-        # softermax n_bias parameter
-        self.n_bias = config.n_bias
+        # MZ: softermax n_bias, clipped softmax n_clip parameters
+        self.n_bias = nn.Parameter(torch.tensor(config.n_bias), requires_grad=config.learn_softmax)
```

Comment:
Will be fixed constants when `require_grad=False` --- cause a Parameter will be no different than a Tensor then.

Reply:
yeah sounds correct 👍